### PR TITLE
Canon.GPCL6_V4_PrinterDriver_V21.00 version 2.72

### PR DESCRIPTION
**Pull request by Vuzeo**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

**Comment by Vuzeo** [comment]
@microsoft-github-policy-service agree company="No company"

**Review by Vuzeo — COMMENTED** [review]
Minor changes

File: manifests/c/Canon/GPCL6_V4_PrinterDriver_V21/00/2.72/Canon.GPCL6_V4_PrinterDriver_V21.00.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Canon.GPCL6_V4_PrinterDriver_V21.00
+PackageVersion: 2.72
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://gdlp01.c-wss.com/gds/0/0100011580/03/GPlus_PCL6_Driver_V272_32_64_00.exe
+  InstallerSha256: D978EE6C4172EA568D6AAE8FD3E874FA2096BF93FF90717E4FE34550FD75E385
```
Comment (Trenly) [inline manifests/c/Canon/GPCL6_V4_PrinterDriver_V21/00/2.72/Canon.GPCL6_V4_PrinterDriver_V21.00.installer.yaml:10]:
```suggestion
  InstallerSha256: D978EE6C4172EA568D6AAE8FD3E874FA2096BF93FF90717E4FE34550FD75E385
  ReleaseDate: 2023-03-16
```

File: manifests/c/Canon/GPCL6_V4_PrinterDriver_V21/00/2.72/Canon.GPCL6_V4_PrinterDriver_V21.00.locale.en-EU.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+PackageVersion: 2.72
+PackageLocale: en-EU
+Publisher: Canon
+PublisherUrl: https://my.canon/en/landing
+PublisherSupportUrl: https://my.canon/en/support
+PrivacyUrl:   https://my.canon/privacy
+PackageName: Canon PCL6 Driver
+PackageUrl: https://my.canon/en/support/0100952701?model=3617B006
+License: MIT License
+ShortDescription: Installation file for Canon PCL6 Drivers version 2.72, dated March 16th, 2023
```
Comment (Trenly) [inline manifests/c/Canon/GPCL6_V4_PrinterDriver_V21/00/2.72/Canon.GPCL6_V4_PrinterDriver_V21.00.locale.en-EU.yaml:14]:
```suggestion
ShortDescription: Installation file for Canon PCL6 Drivers
```
The version and date should be omitted from a static field

**Review by Trenly — CHANGES_REQUESTED** [review]
This package is probably not a good fit for winget as it does not write any values to the registry, so winget will have no way to know that it is installed or that an upgrade may be available

**Review by Vuzeo — COMMENTED** [review]
Agreed.

**Review by Vuzeo — COMMENTED** [review]
Updated information.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**